pkg/asset/templates/kube-controller-manager.yaml

```diff
@@ -5,13 +5,11 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: kube-controller-manager
-    version: v1.3.0-beta.2
```

Comment:
Can we dig into why this is necessary? Not being able to support label metadata seems odd. It's also used in upstream addons - so how are those being upgraded? https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/cluster-monitoring/standalone/heapster-controller.yaml